Reduced the quickstart steps from eight to six (#805)
novuhq/docs: content/docs/platform/quickstart/nextjs.mdx

  </Step>

  <Step>
  ### Add the notification Inbox to your app
    Import Novu’s built-in <Method href="/platform/inbox/react/components/inbox">{`<Inbox />`}</Method> component into your layout file and place it in the navbar:

    ```tsx title="app/layout.tsx"
      import { Inbox } from '@novu/nextjs'; // [!code highlight]
      import './globals.css';

      export const metadata = {
        title: 'Novu Next.js Quickstart',
        description: 'Generated by create next app',
      };

      export default function RootLayout({ children }) {
        return (
          <html lang="en">
            <body>
            // [!code highlight]
              <nav className="flex justify-start items-center p-4 gap-4 h-16"> 
                <Inbox // [!code highlight]
                  applicationIdentifier="YOUR_APPLICATION_IDENTIFIER" // [!code highlight]
                  subscriberId="YOUR_SUBSCRIBER_ID" // [!code highlight]

>                   // [!code highlight]

Reviewer: WE should use the previous inbox code block component, as it auto injects the Keys here for logged in users

                />
                // [!code highlight]
              </nav> 
              {children}
            </body>
          </html>
        );
      }
    ```
    <div className="text-sm text-gray-500 text-center mt-[-12px]">
      [Sign in](https://dashboard.novu.co/auth/sign-up) to get your own API keys
    </div>

    If you are signed in to your Novu account, then the :tooltip[applicationIdentifier]{label="The application identifier is a unique identifier for your application."} and :tooltip[subscriberId]{label="The subscriber ID is the unique identifier for the user in your application, typically the user's id in your database."} will be automatically populated. Otherwise, retrieve them from:

>     * `applicationIdentifier`: On the Novu dashboard, click **API Keys**, and then locate your unique Application Identifier.

Reviewer 2: How? Where is it?

Author: I'm not sure I understand what you mean here. Can you clarify.

Reviewer 2: How do I locate the Application Identifier?

Author: On the Novu dashboard, the user would click "API Keys" on the sidebar, and then locate and copy their unique Application Identifier
<img width="1440" alt="Screenshot 2025-03-31 at 9 48 32 PM" src="https://github.com/user-attachments/assets/eccdf7b9-39db-4f42-ae31-398e79128695" />

>     * `subscriberId` – On the Novu dashboard, click **Subscribers**, and then locate the subscriber ID. For quick start purposes, use the auto-generated subscriber ID for your dashboard user.

Reviewer: I'm not sure I like this, it's not explaining what a subscriber id is. Previously it was better explaining that the subscriber id represents a user on my system

Reviewer 2: Agreed. I look at that and wonder how to locate the subscriber ID as well as wonder what it is..

Reviewer 2: Oh, wow. Honestly, my eyes just skimmed right over the tooltip.

Reviewer: I think that the previous subscriberId text block was better. This might lead the user to think that he needs to do it manually each time in the dashboard, which is not the case.

Author: I see your point, if users think they need to manually retrieve subscriberId from the dashboard every time, that’s a problem. My goal was to provide clarity on where to find it, not imply that manual retrieval is always required.

I will revert to the previous. Something like this.

> (subscriberId) – This represents a user in your system, usually the user ID from your database. For testing, you can use the auto-generated subscriberId from your Novu dashboard.

This keeps the original explanation while making it clear that users don't need to retrieve it manually each time.

    <Callout type="info">
      **Note:** If you pass a `subscriberId` that does not exist yet, Novu will automatically create a new subscriber with that ID.
    </Callout>
    </Step>
  <Step>
  ### Run your application
    Start your development server:

    ```package-install
    npm run dev
    ```

>     Once your application is running, you should see a **bell icon** in the navbar. Clicking it opens the notification inbox UI.

Reviewer 2: Passive: Clicking it opens the...
Active: Click the bell icon to open the...

>     Since there are no notifications yet, let’s trigger one!

Reviewer 2: ```suggestion
    There are no notifications yet, so let’s trigger one!
```

  </Step>

  <Step>
  ### Trigger your first notification
    In this step, you'll create a simple workflow to send your first notification via the Inbox component. Follow these steps to set up and trigger a workflow from your Novu dashboard.
    1. Go to your [Novu dashboard](https://dashboard.novu.co/auth/sign-in).

>     2. Click **Workflows** on the sidebar.

Reviewer 2: Unless there is more than one "Workflows" that the user might click, "on the sidebar" is unnecessary.

>     3. Click **Create Workflow** and enter a name (e.g., "Welcome Notification").

Reviewer 2: ```suggestion
    3. Click **Create Workflow** and then enter a name (e.g., "Welcome Notification").
```

>     4. Click **Create Workflow** to save it. This will automatically open the workflow editor.

Reviewer 2: ```suggestion
    4. Click **Create Workflow** to save it.
        This automatically opens the workflow editor.
```

    5. Click the **Add Step** icon and then select **In-App** as the step type.
    6. In the **In-App template editor**, enter:
        - **Subject**: "Welcome to Novu"
        - **Body**: "Hello, world!"

>     7. Close the editor, then click **Trigger**.

Reviewer 2: A lot of steps like this are actually two steps. I get why you're condensing them for this particular page, but keep an eye out for that. It can confuse the user.

Author: Alright
